pkgs/games/r2modman/default.nix

```diff
@@ -114,5 +114,6 @@ stdenv.mkDerivation rec {
     license = licenses.mit;
     maintainers = with maintainers; [ aidalgol huantian ];
     inherit (electron.meta) platforms;
+    mainProgram = pname;
```

Comment:
The point of mainProgram was not to inherit pname which could have a suffix but to set a literal string.

Reply:
Fixed in https://github.com/NixOS/nixpkgs/pull/275499